### DISCUSSION TITLE
Purchase: Do plugins still work after the purchase period expires?

### DISCUSSION THREAD

**Discussion by dekiesel** [pgmodeler/pgmodeler#1845] [Q&A] [answered]
I am about to buy a package, but I am a bit confused by the options. Will all features work forever for the version I buy or will e.g. plugins stop working after six months?

**Comment by rkhaotix** [comment]
Yes. Once expired the access key you be denied the access to newer versions but you will be allowed to download the original version (fallback version) at the time of purchase. 

For example, let's say that you bought 1.0.6 and you access key expired when 1.2.0 was about to be released, you'll not get 1.2.0 but 1.0.6 will be freely accessible.

**Reply by dekiesel** [reply]
And plugins will continue to work on 1.0.6 but not on 1.2.0, right?

**Comment by rkhaotix** [comment, marked as answer]
Exactly.
